js/DataService.js

```diff
@@ -113,6 +119,7 @@ angular.module('PathOfDamage')
           name: "Damage Mitigation",
           description: "Elemental and chaos damage is mitigated by its respective resistance. Physical damage is mitigated by the sum of all '% additional Physical Damage Reduction' modifiers, up to its 90% cap. This includes armor, endurance charges, and things like Basalt Flasks and Chaos Golem.",
           healthPool: 5000,
+          manaPool: 250,
```

Comment:
People who use MoM on reddit talk about having like 1k unreserved mana, maybe the default should be bigger?